sdk/eventhubs/tests.yml

```diff
@@ -4,21 +4,70 @@ jobs:
-        AZURE_EVENTHUBS_EVENT_HUB_NAME: $(java-event-hubs-event-hub-name)
-        AZURE_TEST_MODE: RECORD
-        AZURE_LOG_LEVEL: 4
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        PRIMARY_STORAGE_ACCOUNT_NAME: $(java-storage-test-primary-account-name)
-        PRIMARY_STORAGE_ACCOUNT_KEY: $(java-storage-test-primary-account-key)
-        SECONDARY_STORAGE_ACCOUNT_NAME: $(java-storage-test-secondary-account-name)
-        SECONDARY_STORAGE_ACCOUNT_KEY: $(java-storage-test-secondary-account-key)
-        BLOB_STORAGE_ACCOUNT_NAME: $(java-storage-test-blob-account-name)
-        BLOB_STORAGE_ACCOUNT_KEY: $(java-storage-test-blob-account-key)
-        PREMIUM_STORAGE_ACCOUNT_NAME: $(java-storage-test-premium-account-name)
-        PREMIUM_STORAGE_ACCOUNT_KEY: $(java-storage-test-premium-account-key)
-        AZURE_STORAGE_ACCESS_KEY: $(java-storage-test-access-key)
+
+        # NB: The PreRunSteps and PostRunSteps are here to prove the concept in the
+        # defined scenario. A script will do this work and we will invoke the script
+        # from the archetype template, so the only changes required will be in
+        # the EnvVars.
+      PreRunSteps:
+          - pwsh: |
+                # $uniqueIdentifier generates a pseudorandom value that represents a unique name that can be used with all resources
+                $uniqueIdentifier =  "t" + [guid]::NewGuid().ToString().Replace("-", "").Substring(0,16)
+                $jobName = "$(Agent.JobName)".Replace(" ", "_")
+                $resourceGroupName = "Tests-$(Build.BuildId)-" + $jobName + "-" + $uniqueIdentifier
+                echo "##vso[task.setvariable variable=ResourceGroupName]$resourceGroupName"
+                echo "##vso[task.setvariable variable=ScopedIdentifier]$uniqueIdentifier"
+                $resourceGroupExpiration = (Get-Date).ToUniversalTime().AddHours(24).ToString("o")
+
+                az login --service-principal -u $env:PROVISIONER_AAD_ID -p $env:PROVISIONER_AAD_SECRET --tenant $env:PROVISIONER_AAD_TENANT
+                az account set --subscription $env:PROVISIONER_SUBSCRIPTION
+
+                az group create --name $resourceGroupName --location $env:PROVISIONER_RG_LOCATION --tags DeleteAfter=$resourceGroupExpiration
+                az group wait --created --resource-group $resourceGroupName
+                $rawDeployOutput = az group deployment create --resource-group $resourceGroupName --template-file sdk/storage/test-resources.json --parameters baseName=$uniqueIdentifier
+
+                $deploymentOutput = $rawDeployOutput | ConvertFrom-JSON -AsHashtable
+                $deploymentOutput.properties.outputs.GetEnumerator() | foreach { Write-Host "##vso[task.setvariable variable=$($_.Name.ToUpper());issecret=true]$($_.value.value)" }
+            env:
```

Comment:
@danieljurek  The pipeline failing because it want a '-' for  env. Can you help me format this in right way so that pipeline consume this file properly.
Or you have a final version to go in this section and we can get rid of this section.

Reply:
This should be resolved by updating `tests.yml` to match what's in `feature/live-test-resource-mgmt` and rebasing on that.